src/konnektor/network_planners/concatenator/cyclic_concatenator.py

```diff
@@ -44,9 +61,36 @@ def concatenate_networks(self, ligand_networks: Iterable[LigandNetwork]) -> Liga
         Returns
         -------
         LigandNetwork
-            returns a concatenated LigandNetwork object, containing all networks.
+            returns a concatenated LigandNetwork object,
+            containing all networks.
 
         """
         raise NotImplementedError()
-
         # Todo: implement.
+
+        selected_edges = []
+        selected_nodes = []
+        for ligandNetworkA, ligandNetworkB in itertools.combinations(
+                ligand_networks, 2):
+            # Generate fully connected Bipartite Graph
+            ligands = list(ligandNetworkA.nodes) + list(ligandNetworkB.nodes)
```

Comment:
since these are sets, could you use `ligandNetworkA.nodes | ligandNetworkB.nodes`

Reply:
always foget those!